politeiawww/pi/pi.go

```diff
@@ -144,3 +143,8 @@ func New(cfg *config.Config, pdc *pdclient.Client, udb user.Database, s *session
 
 	return &p, nil
 }
+
+type mailer interface {
```

Comment:
This should not be redefined.

Reply:
Moved to `mail` package, as suggested in the comments above